RetailCoder.VBE/UI/Command/Refactorings/RefactorExtractMethodCommand.cs

```diff
@@ -77,9 +63,18 @@ protected override void OnExecute(object parameter)
 
             var pane = Vbe.ActiveCodePane;
             var module = pane.CodeModule;
-            var component = module.Parent;
             {
-                Func<QualifiedSelection?, string, IExtractMethodModel> createMethodModel = (qs, code) =>
+                var extraction = new ExtractMethodExtraction();
+                // bug: access to disposed closure - todo: make ExtractMethodRefactoring request reparse like everyone else.
```

Comment:
maybe put the todo in a separate line? And possibly remove it, because it's not a concern for this place, but the `refactoring.Refactor()`